cpp/include/cudf/copying.hpp

```diff
@@ -591,8 +616,8 @@ struct packed_columns {
  * not outlive the memory owned by `data`
  */
 struct packed_table {
-  cudf::table_view table;
-  packed_columns data;
+  cudf::table_view table;  ///< Result table_view of a `contiguous_split`
```

Comment:
```suggestion
  cudf::table_view table;  ///< Result table_view of a cudf::contiguous_split
```
This should add a link to the API in this comment in the rendered doxygen output.